go.mod

```diff
@@ -1,111 +1,110 @@
 module github.com/athanorlabs/atomic-swap
 
-go 1.20
+go 1.21
 
 require (
 	filippo.io/edwards25519 v1.0.0
 	github.com/ChainSafe/chaindb v0.1.6
 	github.com/MarinX/monerorpc v1.0.7
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/athanorlabs/go-dleq v0.1.0
-	github.com/athanorlabs/go-p2p-net v0.3.0
+	github.com/athanorlabs/go-p2p-net v0.3.1-0.20231210033046-13ef86a8318f
 	github.com/btcsuite/btcd/btcutil v1.1.3
-	github.com/cockroachdb/apd/v3 v3.2.0
+	github.com/cockroachdb/apd/v3 v3.2.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
-	github.com/ethereum/go-ethereum v1.12.2
-	github.com/fatih/color v1.15.0
-	github.com/go-playground/validator/v10 v10.15.0
+	github.com/ethereum/go-ethereum v1.13.4
```

Comment:
There is a v1.13.5 of go-ethereum, but it is adversely affecting unit test `TestSwapCreator_Refund_afterT2`.  This PR is for libp2p updates, so I didn't look into why.